packages/webapp/src/components/Crop/Detail.jsx

```diff
@@ -57,7 +64,9 @@ function PureCropDetail({
       <CropHeader
         onBackClick={() => history.back()}
         crop_translation_key={variety.crop_translation_key}
-        crop_variety_name={variety.crop_variety_name}
+        crop_variety_name={variety.crop_varietal}
+        crop_cultivar={variety.crop_cultivar}
+        crop_scientific_name={scientificNameLabel}
         crop_variety_photo_url={variety.crop_variety_photo_url}
         supplier={variety.supplier}
```

Comment:
It seems to me that it is better to send `variety` and destructure it in `CropHeader` like this:
```const { crop_varietal, crop_cultivar, ... } = variety;```

Also, you can create `scientificNameLabel` in `CropHeader` to avoid duplicate code.